lib/screens/app/process_dynamic_links.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:http/http.dart';
+import 'package:provider/provider.dart';
+import 'package:seeds/features/scanner/telos_signing_manager.dart';
+import 'package:seeds/providers/notifiers/settings_notifier.dart';
+import 'package:seeds/providers/services/esr_service.dart';
+import 'package:seeds/providers/services/links_service.dart';
+import 'package:seeds/screens/app/wallet/custom_transaction.dart';
+import 'package:url_launcher/url_launcher.dart' as UrlLauncher;
+import 'package:convert/convert.dart';
+
+class ProcessDynamicLinks extends StatefulWidget {
+  final Widget child;
+
+  ProcessDynamicLinks({@required this.child});
+
+  @override
+  _ProcessDynamicLinksState createState() => _ProcessDynamicLinksState();
+}
+
+class _ProcessDynamicLinksState extends State<ProcessDynamicLinks> {
+  @override
+  void initState() {
+    super.initState();
+    processSigningRequests();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return widget.child;
+  }
+
+  void processIdentityRequest(SeedsESR request) async {
+    var defaultPermission = {"actor": SettingsNotifier.of(context, listen: false).accountName, "permission": "active"};
+
+    bool doesNotHavePermission =
+        request.manager.data.req[1] == null || request.manager.data.req[1]["permission"] == null;
+
+    if (doesNotHavePermission) {
+      request.manager.data.req[1]["permission"] = defaultPermission;
+    }
+
+    var confirmed = await showDialog(
+        context: context,
+        builder: (_) {
+          return AlertDialog(
+            title: Text("Confirm Authorization"),
+            actions: [
+              FlatButton(
+                child: Text("Cancel"),
+                onPressed: () => Navigator.of(context).pop(),
+              ),
+              FlatButton(
+                child: Text("Confirm"),
+                onPressed: () => Navigator.of(context).pop(true),
+              ),
+            ],
+          );
+        });
+
+    if (confirmed == true) {
+      try {
+        var response = await EsrService().getIdentityResponse(
+          request: request,
+          accountName: SettingsNotifier.of(context, listen: false).accountName,
+          walletPrivateKey: SettingsNotifier.of(context, listen: false).privateKey,
+        );
+
+        var theresponse = await post(
+          response.callback,
+          body: response.body,
+        );
+
+        print(theresponse);
+      } catch (err) {
+        print(err);
+      }
+    }
+  }
+
+  void processTransactionRequest(SeedsESR request) async {
+    await request.resolve(
```

Comment:
we no longer use `SettingsNotifier` to get the accountName
we use the storageService i think its the name. Its a singleton